cli/nbd-cli.c

```diff
@@ -118,11 +118,11 @@ static int load_our_module(void)
         state = kmod_module_get_initstate(mod);
         switch (state) {
         case KMOD_MODULE_BUILTIN:
-            nbd_out("Module '%s' is builtin\n", kmod_module_get_name(mod));
+            nbd_info("Module '%s' is builtin\n", kmod_module_get_name(mod));
```

Comment:
Let's make this as DEBUG info, and the same as the followings.

Reply:
OK